[2023-03-21 07:06:27,394] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: querry_postgres_v1.1.insert_querry_postgres_table manual__2023-03-21T07:05:45.710019+00:00 [queued]>
[2023-03-21 07:06:27,436] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: querry_postgres_v1.1.insert_querry_postgres_table manual__2023-03-21T07:05:45.710019+00:00 [queued]>
[2023-03-21 07:06:27,438] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-21 07:06:27,439] {taskinstance.py:1239} INFO - Starting attempt 2 of 3
[2023-03-21 07:06:27,440] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-21 07:06:27,487] {taskinstance.py:1259} INFO - Executing <Task(PostgresOperator): insert_querry_postgres_table> on 2023-03-21 07:05:45.710019+00:00
[2023-03-21 07:06:27,493] {standard_task_runner.py:52} INFO - Started process 33406 to run task
[2023-03-21 07:06:27,501] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'querry_postgres_v1.1', 'insert_querry_postgres_table', 'manual__2023-03-21T07:05:45.710019+00:00', '--job-id', '254', '--raw', '--subdir', 'DAGS_FOLDER/querry_posgresql.py', '--cfg-path', '/tmp/tmp7te89pe7', '--error-file', '/tmp/tmp9_zg8r32']
[2023-03-21 07:06:27,504] {standard_task_runner.py:77} INFO - Job 254: Subtask insert_querry_postgres_table
[2023-03-21 07:06:27,824] {logging_mixin.py:109} INFO - Running <TaskInstance: querry_postgres_v1.1.insert_querry_postgres_table manual__2023-03-21T07:05:45.710019+00:00 [running]> on host 0b4cdb459521
[2023-03-21 07:06:28,147] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=sakabuana31
AIRFLOW_CTX_DAG_ID=querry_postgres_v1.1
AIRFLOW_CTX_TASK_ID=insert_querry_postgres_table
AIRFLOW_CTX_EXECUTION_DATE=2023-03-21T07:05:45.710019+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-21T07:05:45.710019+00:00
[2023-03-21 07:06:28,227] {base.py:79} INFO - Using connection to: id: postgres_localhost. Host: host.docker.internal, Port: 5432, Schema: final, Login: ***, Password: ***, extra: {}
[2023-03-21 07:06:28,309] {dbapi.py:225} INFO - Running statement: 
            INSERT INTO serving_table_review (date, status_temp, status_preci, business_id, business_name, stars, date_values)
            SELECT review.date_values, temperature.status_temp, precipitation.status_preci, review.business_id, business.name, review.stars, review.date_values
            FROM review
            LEFT JOIN precipitation ON (review.date_values = precipitation.date)
            LEFT JOIN temperature ON (review.date_values = temperature.date)
            LEFT JOIN business ON (review.business_id = business.business_id)
            ORDER BY review.date_values
        , parameters: None
[2023-03-21 07:06:28,901] {dbapi.py:233} INFO - Rows affected: 72125
[2023-03-21 07:06:29,090] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=querry_postgres_v1.1, task_id=insert_querry_postgres_table, execution_date=20230321T070545, start_date=20230321T070627, end_date=20230321T070629
[2023-03-21 07:06:29,180] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-21 07:06:29,403] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
